app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/components/PlanWizardInstancePropertiesStepTable.js

```diff
@@ -36,40 +50,62 @@ class PlanWizardInstancePropertiesStepTable extends React.Component {
 
   // enables our custom header formatters extensions to reactabular
   customHeaderFormatters = Table.customHeaderFormattersDefinition;
-
-  // TODO:  These methods need to be wired up to redux-form once we know what
-  //        the POST body should look like for creating an OSP transformation
-  //        plan.  We will use local state to store the row that is currently
-  //        being edited to roll back to in case of cancel.
   inlineEditController = () => {
-    const { rows } = this.props;
+    const { rows, instancePropertiesRowsAction, input } = this.props;
     return {
-      isEditing: ({ rowData }) => rowData.id === this.state.backup.id,
+      isEditing: ({ rowData }) => rowData.id === input.value.updatedInstancePropertiesRowOnStandby.id,
       onActivate: ({ rowData }) => {
-        const index = findIndex(rows, { id: rowData.id });
-        const backup = rows[index];
-
-        this.setState({ backup, editing: true });
+        this.setState({ editing: true });
+        input.onChange({ updatedInstancePropertiesRowOnStandby: rowData });
       },
       onConfirm: ({ rowData }) => {
-        this.setState({ backup: {}, editing: false });
+        this.setState({ editing: false });
+
+        const updatedRows = rows.map(
+          row => (row.id === rowData.id ? input.value.updatedInstancePropertiesRowOnStandby : row)
+        );
+        instancePropertiesRowsAction(updatedRows);
+        input.onChange({ updatedInstancePropertiesRowOnStandby: {} });
       },
       onCancel: ({ rowData }) => {
-        this.setState({ backup: {}, editing: false });
+        this.setState({ editing: false });
+        input.onChange({ updatedInstancePropertiesRowOnStandby: {} });
       },
-      onChange: (value, { rowData, property }) => {}
+      onChange: (e, { rowData, property }) => {
+        let updatedRowdata = '';
+        const updatedInstanceProp = {
+          ...rowData[property],
+          name: e.target.options[e.target.selectedIndex].text,
+          id: e.target.value
+        };
+
+        if (property === 'osp_security_group') {
+          updatedRowdata = {
+            ...input.value.updatedInstancePropertiesRowOnStandby,
+            osp_security_group: updatedInstanceProp
+          };
+        } else if (property === 'osp_flavor') {
+          updatedRowdata = { ...input.value.updatedInstancePropertiesRowOnStandby, osp_flavor: updatedInstanceProp };
+        }
```

Comment:
Oops, sorry, I should have caught this when I made [comment](https://github.com/ManageIQ/manageiq-v2v/pull/627#discussion_r215941040)

This conditional block can be simplified to

```js
updatedRowdata = {
  ...input.value.updatedInstancePropertiesRowOnStandby,
  [property]: updatedInstanceProp
};
```

Also, up in line 75 can we please initialize `updatedRowdata` with an empty object :)

Reply:
Nice!
I had actually intended to make it generic, so thanks for the above suggestion!!